### PR TITLE
review: Allow to use spreadsheet without credentials file

### PR DESCRIPTION
**Pull request by surli**
The idea of this PR is to be able to use a launcher to create credentials file, and then to be able to use only an access_token to obtain credentials, to be able to pass it easily to docker containers.

### CODE REVIEW

**Comment by surli** [comment]
@fermadeiral it's ready for review!

**Comment by fermadeiral** [comment]
> @fermadeiral it's ready for review!

Ok.

File: repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java [hunk 1]
```diff
@@ -209,16 +232,11 @@ private void mainProcess() throws IOException {
...
-            serializers.add(new GoogleSpreadSheetInspectorTimeSerializer(this.config.getGoogleSecretPath()));
-            serializers.add(new GoogleSpreadSheetNopolSerializer(this.config.getGoogleSecretPath()));
+            serializers.add(new GoogleSpreadSheetInspectorSerializer());
+            serializers.add(new GoogleSpreadSheetInspectorTimeSerializer());
+            serializers.add(new GoogleSpreadSheetNopolSerializer());
         } else {
-            GoogleSpreadSheetFactory.setSpreadsheetId(GoogleSpreadSheetFactory.BEAR_SPREADSHEET_ID);
-
-            serializers.add(new GoogleSpreadSheetInspectorSerializer4Bears(this.config.getGoogleSecretPath()));
-            serializers.add(new GoogleSpreadSheetInspectorTimeSerializer4Bears(this.config.getGoogleSecretPath()));
```
Comment (fermadeiral) [inline repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java]: I think you put `GoogleSpreadSheetInspectorTimeSerializer4Bears` out of work.

Comment (surli) [inline repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java]: Hmmm you're right, I did not pay enough attention when merging, I'll fix that.

Comment (surli) [inline repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java]: Done.

**Pull request merged**